tests/client/amo/containers/TestApp.js

```diff
@@ -137,24 +130,6 @@ describe('App', () => {
     assert.ok(dispatch.calledWith({ type: INSTALL_STATE, payload }));
   });
 
-  it('renders redux-connect errors', () => {
-    // This is just a sanity check to make sure the default component
-    // is wrapped in handleResourceErrors
-    const store = createStore();
-    const apiError = api.createApiError({
-      apiURL: 'https://some-url',
-      response: { status: 404 },
-    });
-    store.dispatch(reduxConnectLoadFail('someKey', apiError));
-
-    const root = renderIntoDocument(
-      <WrappedApp store={store} />
-    );
-
-    const rootNode = findDOMNode(root);
-    assert.include(rootNode.textContent, 'Not Found');
```

Comment:
Since we're not using Flow to guarantee that `ErrorPage` is what you say it is, it would make sense to include a test that looks like this. Otherwise, it's hard to make sure that `ErrorPage` is configured in the App component correctly. It would be sort of like a "smoke test."